docs/kbs.yaml

```diff
@@ -0,0 +1,267 @@
+          content:
+            headers:
+              Set-Cookie:
+                name: kbs-session-id
+                schema:
+                  type: string
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResponseHeader'
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ResponsePayload'
+        401:
+          description: Missing or invalid session ID
+        403:
+          description: The KBC is not allowed to get that key
+        404:
+          description: The requested key does not exist
+
+  /resources/token/:
+    get:
+      summary: Get an external resource token from the Key Broker Service
+      parameters:
+        in: cookie
+        name: kbs-session-id
+        schema:
+          type: string
+      responses:
+        200:
+          description: The Key Broker reponse including the token
+          content:
+            headers:
+              Set-Cookie:
+                name: kbs-session-id
+                schema:
+                  type: string
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TokenHeader'
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TokenPayload'
+        401:
+          description: Missing or invalid session ID
+        403:
+          description: The KBC is not allowed to get a token
+        404:
+          description: The KBS does not provide token resources
+
+components:
+  schemas:
+
+    Request:
+      required:
+        - version
+        - tee
+      type: object
+      properties:
+        version:
+          type: string
+        tee:
+          type: string
+          enum: [intel-tdx, intel-sgx, amd-sev]
+        extra-params:
+          description:
+            Freely formatted JSON object used for HW-TEE specific attestation
+            processes.
+          type: object
+      description: KBC attestation authentication request
+
+    Challenge:
+      required:
+        - nonce
+      type: object
+      properties:
+        nonce:
+          type: string
+        extra-params:
+          description:
+            Freely formatted JSON object used for HW-TEE specific attestation
+            processes.
+          type: object
+      description:
+        KBS attestation challenge, sent in response to the KBC Request.
+
+    Attestation:
+      required:
+        - tee-pubkey
+        - tee-evidence
+      type: object
+      properties:
+        tee-pubkey:
+          $ref: "#/components/schemas/PublicKey"
+        tee-evidence:
+          description:
+          type: object
+      description:
+        KBS attestation challenge, sent in response to the KBC Request.
+
+    ResponseHeader:
+      required:
+        - alg
+        - enc
+      type: object
+      properties:
+        alg:
+          type: string
+        enc:
+          type: string
```

Comment:
Does OpenAPI allow you to add a description for `alg` and `enc` as they are brutally terse ;)